src/components/Explore/Sidebar.tsx

```diff
@@ -44,12 +45,18 @@ const Sidebar = () => {
                         ]}
                     />
                     <Box sx={{ alignSelf: 'center', zIndex: 1 }}>
-                        <Button
-                            variant="outlined"
-                            onClick={() => dataActionDispatcher({ type: 'updateSelectedStation', station: null })}
-                        >
-                            Go Back
-                        </Button>
+                        <Stack direction="column" spacing={4}>
```

Comment:
Why are these needed? Looks like they can be removed?
Here's another thought: can we move the download button next to the "Go Back" button? This way it stays visible on all tabs.

Reply:
So the problem is I am stringifying the `stationDetails` object so on the upper component, we don't have the details yet. That is why I move it inside. one thing is we can move the state of `stationDetails` up and then add the download button there.